corehq/apps/users/views/mobile/users.py

```diff
@@ -1067,6 +1069,11 @@ def post(self, request, *args, **kwargs):
             list(self.group_specs),
             request.couch_user
         )
+        upload_record = UserUploadRecord(
+            domain=self.domain,
+            task_id=task.id,
+        )
+        upload_record.save()
```

Comment:
is there an advantage of creating this object here and not in `import_users_and_groups` itself?

Reply:
I wanted it here so that it was created synchronously, rather than waiting for celery to get spun off. That way we have a record as soon as an attempt is made, even in the presence of celery issues